dbt/models/shared_columns.md

```diff
@@ -0,0 +1,1085 @@
+{% enddocs %}
+
+## pin_num_cards
+
+{% docs shared_column_pin_num_cards %}
+Number of cards associated with a PIN.
+
+PINs can have multiple cards (buildings) within their area. Multi-card PINs
+are valued slightly differently than single-card ones.
+{% enddocs %}
+
+## pin_num_landlines
+
+{% docs shared_column_pin_num_landlines %}
+Number of landlines associated with a PIN.
+
+Each landlines is typically associated with an individual land rate i.e.
+rate varies within the PIN. The sum of all landline square footage _should_
+be equal to the total square footage of the parcel. Each landline can
+correspond to a different land price/rate. NOTE: there are cases where land
+itself is prorated, in which case landline values may not sum to the parcel
+square footage
+{% enddocs %}
+
+## pin_num_lline
+
+{% docs shared_column_pin_num_lline %}
+Number of `lline` values associated with a PIN
+{% enddocs %}
+
+## tieback_key_pin
+
+{% docs shared_column_tieback_key_pin %}
+Key PIN for prorated PINs.
+
+See `tieback_proration_rate` for explanation of PIN-level proration.
+{% enddocs %}
+
+## tieback_proration_rate
+
+{% docs shared_column_tieback_proration_rate %}
+Proration rate applied to the PIN.
+
+PINs are occasionally prorated when not all of their value is contained within
+their boundaries. For example, a building that lies equally across two PINs
+would be prorated to 50%. In this case, the _land_ value of the PIN is not
+prorated, but the building value is.
+{% enddocs %}
+
+# Sales
+
+## buyer_name
+
+{% docs shared_column_buyer_name %}
+Name of property buyer, as listed on deed.
+
+Can be truncated by myDec/IDOR. See Clerk/Recorder of Deeds for full name.
+{% enddocs %}
+
+## document_number
+
+{% docs shared_column_document_number %}
+Deed number/document number of the sale.
+
+Serves as the unique identifier for each sale. Can be used to lookup more
+information on the Clerk/Recorder of Deeds website.
+{% enddocs %}
+
+## deed_type
+
+{% docs shared_column_deed_type %}
+Numerically-encoded deed type of the sale.
+
+Possible values for this variable include:
+
+- `01` - Warranty
+- `02` - Trustee
+- `03` - Quit claim
+- `04` - Executor
+- `05` - Other
+- `06` - Beneficiary
+- `99` - Unknown
+{% enddocs %}
+
+## num_parcels_sale
+
+{% docs shared_column_num_parcels_sale %}
+Number of parcels that are part of multisale
+{% enddocs %}
+
+## sale_date
+
+{% docs shared_column_sale_date %}
+Date of property sale.
+
+Uses dates from `sale.mydec` prior to 2021. Also defaults to `sale.mydec`
+any time `iasworld.sales` and `sale.mydec` disagree.
+{% enddocs %}
+
+## sale_is_multisale
+
+{% docs shared_column_sale_is_multisale %}
+Indicates whether or not a sale involved multiple parcels
+{% enddocs %}
+
+## sale_key
+
+{% docs shared_column_sale_key %}
+iasWorld internal sale identifier
+{% enddocs %}
+
+## sale_price
+
+{% docs shared_column_sale_price %}
+Fair market sale price of a PIN, as recorded on the deed.
```

Comment:
We don't actually know that a random value of sale_price is fair market, do we?

Reply:
True, I'll update the language.